src/View/Widgets/LocationBar.vala

```diff
@@ -103,7 +103,6 @@ namespace Files.View.Chrome {
 
         private void on_search_results_exit (bool exit_navigate = true) {
             /* Search result widget ensures it has closed and released grab */
-            search_results.is_active = false;
```

Comment:
Why remove this? Is there a risk of this being followed by `after_bread_focus_out_event` and causing undefined behavior?

Reply:
Its because SearchResults.is_active is now readonly.

Reply:
I'll check the logic wrt focus_out event though - its hard to remember the details now.

Reply:
SearchResults sets itself inactive *before* emitting the `exit` signal and this is handled before the `after_bread_focus_out_event` so I do not think there is an issue of undefined behaviour.  Tbh the pathbar/searchresults code is very difficult to follow and is likely to be re-written with new widgets for the Gtk4 port.